src/ontology/mondo.Makefile

```diff
@@ -289,6 +289,41 @@ sources/$(SOURCE_VERSION)/equivalencies.owl: | source_release_dir
 #sources/CTD_diseases.obo:
 #	curl -L -s http://ctdbase.org/reports/CTD_diseases.obo.gz  | gzip -dc | perl -npe 's@alt_id@xref@' > $@.tmp && mv $@.tmp $@
 
+# TODO: Gard mapped obsoletes
+# Working helper commands ------
+tmp/mondo-edit.owl:
+	robot convert -i mondo-edit.obo --format owl -o $@
+tmp/mondo-edit.db: tmp/mondo-edit.owl
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	@rm -f .template.db
+	@rm -f .template.db.tmp
+tmp/obsoletes.tsv: tmp/mondo-edit.db
+	runoak -i sqlite:tmp/mondo-edit.db obsoletes > tmp/obsoletes.tsv
+
+ofn:
+	robot convert -i mondo-edit.obo --format ofn -o tmp/mondo-edit.ofn
+
+# Failed commands ------
+#assocs:  # not what I needed
+#	runoak -i sqlite:tmp/mondo-ingest.db associations -p i DOID:2757
+#axioms:  # notImplementedError
```

Comment:
I didn't know if `runoak axioms` would be helpful, but I tried it. I got 

```
runoak -i tmp/mondo-edit-temp.ofn axioms –axiom-type MONDO:equivalentTo
warnings.warn(
.........k.........k.........k.........k.........k.........k.........k.........k.........k.........10000
.........k.........k.........k.........k.........k.........k.........k.........k.........k.........20000
.........k.........k.........k.........k.........k.........k.........k.........k.........k.........30000
.........k.........k........NotImplementedError
```

same result with `–axiom-type subClassOf` per the docs as well.

Reply:
Always run with `tmp/mondo.db`